### DISCUSSION TITLE
chatgpt-web错误, Failed to get arkose token.

### DISCUSSION THREAD

**Discussion by Sunrepe** [nextai-translator/nextai-translator#1493] [Q&A]
![Snipaste_2024-04-29_00-11-48](https://github.com/openai-translator/openai-translator/assets/24914256/627052b0-c348-4812-980e-52f44cfbfc8e)
你好，我已经试过[issue](https://github.com/openai-translator/openai-translator/discussions/1394)中所提出的解决方法
包括：
1, 清楚历史数据
2，注销chatgpt账号，包括chat以及playground中的账号
3，重新登录并与chatgpt进行互动

经过以上上个步骤，报错还是维持着不能获得arkose token。
猜测可能是一个bug，原因：
我在退出openai账号以后，重新点击login会跳转为
类似以下的网址：
https://auth.openai.com/authorize?client_id=TdJIcbe16WoTHtN95nyywh5E4yOo6ItG&scope=openid%20email%20profile%20offline_access%20model.request%20model.read%20organization.read%20organization.write&response_type=code&redirect_uri=https%3A%2F%2Fchat.openai.com%2Fapi%2Fauth%2Fcallback%2Flogin-web&audience=https%3A%2F%2Fapi.openai.com%2Fv1&device_id=8859-7b924a17&prompt=login&screen_hint=login&state=3XRqEF7fjul3bQ&code_challenge=EVMjuJmF-9Lfj_ibrE2ojkg&code_challenge_method=S256

同事，[platform](https://platform.openai.com/playground?mode=complete)和[chat](https://chat.openai.com/)两个网页数据并无法互通，需要分别登录

希望开发人员帮忙看看

**Comment by zhl111** [comment]
你的问题解决了吗？我也是这个问题
